frontends/web/src/routes/account/summary/accountssummary.tsx

```diff
@@ -213,13 +228,16 @@ export function AccountsSummary({ accounts, devices }: TProps) {
                   <AddBuyReceiveOnEmptyBalances accounts={accounts} balances={balances} />
                 ) : undefined
               } />
-            {accountsByKeystore.length > 1 && (
+            {showTotalCoins && (
               <CoinBalance
                 accounts={accounts}
                 summaryData={summaryData}
                 coinsBalances={coinsTotalBalance}
               />
             )}
+            {hasLightningFromOtherKeystore && (
+              <LightningBalance/>
+            )}
```

Comment:
is my understanding correct, that if the Lightning account is from a connected keystore it will be included together with normal accounts?

Reply:
yes, it will be included together with normal accounts

Reply:
it will be included with normal accounts also when it is from a remembered keystore